scripts/issue/646/trops_cleanup_q4.py

```diff
@@ -0,0 +1,59 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    trops = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    trops.init_badger()
+    trops.init_curve()
+
+    cvx = trops.contract(r.treasury_tokens.CVX)
+    bvecvx = trops.contract(r.treasury_tokens.bveCVX)
+    cvxcrv = trops.contract(r.treasury_tokens.cvxCRV)
+    bcvxcrv = trops.contract(r.treasury_tokens.bcvxCRV)
+    gravi = trops.contract(r.sett_vaults.graviAURA)
+    crv3pool = trops.contract(r.treasury_tokens.crv3pool)
+    usdc = trops.contract(r.treasury_tokens.USDC)
+    baurabal = trops.contract(r.sett_vaults.bauraBal)
+
+    send_to_vault = [cvxcrv, bcvxcrv, baurabal]
+
+    trops.take_snapshot(
+        tokens=[bvecvx, bcvxcrv, gravi, usdc, baurabal, cvx] + send_to_vault
+    )
+    vault.take_snapshot(tokens=send_to_vault)
+
+    # receive bvecvx, bcvxCRV, graviAURA
+    trops.badger.claim_all()
+
+    # withdraw all 3crv to usdc
+    trops.curve.withdraw_to_one_coin(crv3pool, crv3pool.balanceOf(trops), usdc)
+
+    # deposit cvx
+    cvx.approve(bvecvx, cvx.balanceOf(trops))
+    bvecvx.deposit(cvx.balanceOf(trops))
+
+    # transfers to vault
+    for token in send_to_vault:
+        token.transfer(vault, token.balanceOf(trops))
```

Comment:
same here; approve and bcvxcrv.depositfor(tree, balanceof(trops)) instead of a direct transfer of the cvxcrv.

also note that the bcvxcrv has to go to the tree in order to get rid of the deficit there, so at some point we can close #620

Reply:
adjusted in 04d79a3, remaining balance of bcvxcrv from trops is sent to tree to as well